corehq/apps/events/models.py

```diff
@@ -210,26 +210,24 @@ def get_attended_attendees(self):
     def _get_attendee_cases_related_to(self, case_type):
         # Attendee cases are associated with one or more Events using
         # extension cases. The extension cases have case types
-        # EVENT_ATTENDEE_CASE_TYPE ('commcare-potential-attendee').
-        # ATTENDEE_DATE_CASE_TYPE ('commcare-attendance-date')
+        # EVENT_ATTENDEE_CASE_TYPE ('commcare-potential-attendee') or
+        # ATTENDEE_DATE_CASE_TYPE ('commcare-attendance-date').
         #
         # The extension cases are owned by the Event's case-sharing
         # group so that all mobile workers in the group get the attendee
         # cases for the Event.
 
         # CommCareCaseIndex.objects.get_extension_case_ids only supports
         #   fetching by exclude_for_case_type, so fetch by exclusion
-        extension_types = {ATTENDEE_DATE_CASE_TYPE, EVENT_ATTENDEE_CASE_TYPE}
-        assert case_type in extension_types, f"This method only supports {extension_types}"
-        exclude_case_type = (extension_types - {case_type}).pop()
-        event_attendee_cases = self._get_ext_cases(exclude_case_type)
+        ext_cases = self._get_ext_cases()
 
         attendee_case_type = get_attendee_case_type(self.domain)
         attendee_cases = []
-        for case in event_attendee_cases:
-            for index in case.indices:
-                if index.referenced_type == attendee_case_type:
-                    attendee_cases.append(index.referenced_case)
+        for case in ext_cases:
+            if case.type == case_type:
```

Comment:
If we are only interested in extension cases with `case_type`, why not do a `self._get_ext_cases(case_type)`? If the helper function should not always require a `case_type`, then it could be defined as `def _get_ext_cases(self, case_type=None)` so that it works for getting a specific case type and all case types.

Reply:
Yes, of course! Thank you!

Reply:
461f3456fb3dce25b54ca9397ebdf003bd1e37b1